docs/deploy.rst

```diff
@@ -0,0 +1,244 @@
+.. _deploy:
+
+
+=========================
+Deploy your first cluster
+=========================
+
+This page shows you how to:
```

Comment:
this para/list is too literal. we should have a short para here that continues the narrative of the getting started tutorial, i.e., "now you have x and y, let's do z"